test/Tests.Infrastructure/CustomS3RetryTheoryAttribute.cs

```diff
@@ -40,27 +40,22 @@ static CustomS3RetryTheoryAttribute()
         }
     }
 
+    public CustomS3RetryTheoryAttribute()
+    {
```

Comment:
where this is used?

Reply:
I added this so in case we will want to pass it to `MultiTheoryAttribute` it will compile since it need default ctor, so `Activator.CreateInstance(t)` won't throw.